eng/pipelines/templates/pipeline.yml

```diff
@@ -79,142 +106,395 @@ stages:
       BuildNumber: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.BuildNumber']]
       FullVstsBuildNumber: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.FullVstsBuildNumber']]
       VsTargetChannel: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.VsTargetChannel']]
+      VsTargetChannelForTests: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.VsTargetChannelForTests']]
       VsTargetMajorVersion: $[stageDependencies.Initialize.Initialize_Build.outputs['updatebuildnumber.VsTargetMajorVersion']]
-      SDKVersionForBuild: $[stageDependencies.Initialize.Initialize_Build.outputs['getSDKVersionForBuild.SDKVersionForBuild']]
+      isOfficialBuild: ${{ parameters.isOfficialBuild }}
+      ShouldSkipOptimize: ${{ variables['ShouldSkipOptimize'] }}
       LocalizedLanguageCount: "13"
       BuildRTM: "false"
+      SemanticVersion: $[stageDependencies.Initialize.GetSemanticVersion.outputs['setsemanticversion.SemanticVersion']]
     pool:
       name: VSEngSS-MicroBuild2019-1ES
+      os: windows
+      demands:
+      - ImageOverride -equals server2022-microbuildVS2019-1es
+    templateContext:
+      mb:
+        localization:
+          enabled: true
+        signing:
+          enabled: true
+          signType: "${{ parameters.SigningType }}"
+          ${{ if and(eq(parameters.isOfficialBuild, true), eq(parameters.SigningType, 'Real')) }}:
+            signWithProd: true
+        swix:
+          enabled: true
+        optprof:
+          enabled: false
```

Comment:
Build_And_UnitTest.yml has removed `- task: MicroBuildOptProfPlugin@6`, so this needs to be enabled.

Reply:
Done, but will require follow-ups in https://github.com/NuGet/Client.Engineering/issues/3374